addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -40,6 +40,7 @@ class Im_LivechatReportChannel(models.Model):
     partner_id = fields.Many2one('res.partner', 'Operator', readonly=True)
     handled_by_bot = fields.Integer("Handled by Bot", readonly=True, aggregator="sum")
     handled_by_agent = fields.Integer("Handled by Agent", readonly=True, aggregator="sum")
+    chatbot_script_id = fields.Many2one("res.partner", "Chatbot", readonly=True)
```

Comment:
Don't we want to make it a many if we already have in mind or have the possibility to have multiple scripts per session?